### PR TITLE
Add unit tests for com.nepxion.discovery.common.entity.InstanceEntity…

### PR DESCRIPTION
**Pull request by Braavos6**
I've analysed your codebase and noticed that com.nepxion.discovery.common.entity.InstanceEntityWrapper is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important.

### CODE REVIEW

**Review by JohnLBergqvist — APPROVED** [review]

**Review by petersear — APPROVED** [review]

File: discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+import static com.nepxion.discovery.common.entity.InstanceEntityWrapper.*;
+
+public class InstanceEntityWrapperTest {
+
+    private InstanceEntity instanceEntity = new InstanceEntity();
+    private HashMap<String, String> hashMap = new HashMap<>();
```
Comment (peterschrammel) [inline discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java]: 🚫 That makes the tests dependent.

File: discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+import static com.nepxion.discovery.common.entity.InstanceEntityWrapper.*;
+
+public class InstanceEntityWrapperTest {
+
+    private InstanceEntity instanceEntity = new InstanceEntity();
```
Comment (peterschrammel) [inline discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java]: make `final`

File: discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    @Test
+    public void testGetContextPath() {
+        assertNull(getContextPath(instanceEntity));
+    }
+
+    @Test
+    public void testGetGroup() {
+        assertEquals("", getGroup(instanceEntity));
```
Comment (peterschrammel) [inline discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java:25]: Would prefer a non-empty group

**Review by peterschrammel — CHANGES_REQUESTED** [review]

File: discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java (hunk 2 above)
Comment (Braavos6) [reply discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java]: done

File: discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java (hunk 1 above)
Comment (Braavos6) [reply discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java]: Removed the global variable and moved it into the methods

File: discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java (hunk 3 above)
Comment (Braavos6) [reply discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java:25]: I tested for a non-empty group in line 35.
The method will only return content if the value "spring.application.group.key" exists else it returns ""

**Review by peterschrammel — APPROVED** [review]

**Pull request closed without merging**